utils/gyb.py

```diff
@@ -1042,18 +1042,19 @@ def execute_template(
     ... THIS SHOULD NOT APPEAR IN THE OUTPUT
     ... ''')
     >>> out = execute_template(ast, line_directive='//#sourceLocation', x=1)
-    >>> out = out.replace(os.path.abspath(os.sep) + 'dummy.file', "DUMMY-FILE")
+    >>> working_directory = os.getcwd() + os.sep
+    >>> out = out.replace(working_directory, "WORKING-DIRECTORY/")
```

Comment:
If line-directive is going to inject absolute paths, we should be passing an absolute path to `parse_template`, as we originally were.